packages/playwright-core/src/web/traceViewer/ui/snapshotTab.tsx

```diff
@@ -65,6 +65,8 @@ export const SnapshotTab: React.FunctionComponent<{
         const info = await response.json();
         if (!info.error)
           setSnapshotInfo(info);
+      } else {
+        setSnapshotInfo(defaultSnapshotInfo);
```

Comment:
If an error occurred when a trace was already loaded before, this cleans up the old snapshot URL which was shown.

Reply:
Put it in as a comment!